cypress/e2e/article.cy.js

```diff
@@ -1,13 +1,42 @@
-describe('', () => {
-  before(() => {
+describe('Article Flow with Pre-set Auth Token', () => {
+  // eslint-disable-next-line max-len
+  const authToken = 'eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpZCI6NjgxNTM0LCJ1c2VybmFtZSI6InRlc3R1c2VyNjAwNiIsImV4cCI6MTczNDI3MTc3MiwiaWF0IjoxNzI5MDg3NzcyfQ.i5DTxajO1KFWmD6UGuHp5hW6P4JhgyVM9OPFNG3d1kE';
 
-  });
+  beforeEach(() => {
+    cy.clearLocalStorage(); cy.clearCookies();
 
-  it('', () => {
+    cy.setCookie('auth', authToken);
```

Comment:
It's a good practice to add a comment explaining why you're setting a cookie with a pre-set auth token. This helps other developers understand the context of your tests.